sale_order_line_sequence/views/sale_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <field name="order_line" position="before">
+                <field name="max_line_sequence" invisible="1" />
```

Comment:
Fields like this, that will never be visible and are only used in attributes of other fields are now added automatically in Odoo 18. I think we can consider removing them from our views for a slightly cleaner layout. What do you think?

I only added this to the migration guidelines yesterday, so you will not have seen it before.
> Missing fields used in domains and other attributes are added to the view as invisible automatically. If you encounter fields in views with invisible="True", they can probably be removed. Reference: https://github.com/odoo/odoo/pull/137031